src/component/Button.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class Button extends Component {
+  constructor(props) {
+    super(props);
+    this.name = props.name;
+    this.value = props.value;
+    this.wide = props.wide;
+    /* this.color = props.color; */
+
+    if (props.wide) {
+      this.width = '50%';
+    }
+
+    if (props.color) {
+      this.color = props.color;
+    }
+
+    this.style = {
+      width: this.width,
+      'background-color': this.color,
```

Comment:
I'm curious, were you trying to write `'background-color'` or `background-color`?
I think it should be without quote right